Automattic/wp-calypso
# Earn: update copy and icon on the Payments splash page in Earn.

This PR updates the copy and icon used on the dedicated Payments page that is shown when who has not yet connected to Stripe clicks the Payments card link in the Earn section of Calypso. Context for the changes is at pbMlHh-i6-p2.

Here are screenshots showing the changes:

Before:
<img width="1068" alt="Screen Shot 2020-06-23 at 9 24 00 PM" src="https://user-images.githubusercontent.com/35781181/85486072-eb43ce00-b597-11ea-9790-fad82684b980.png">

After:
<img width="1072" alt="Screen Shot 2020-06-23 at 3 39 25 PM" src="https://user-images.githubusercontent.com/35781181/85486088-f565cc80-b597-11ea-9ec1-7ce4d5d29f1d.png">


#### Testing instructions

* Checkout this branch (git checkout update/earn-payments-splash-page-copy-edits)
* Start calypso (yarn start)
* Navigate to the Earn page using an account with a paid plan that has not connected to Stripe (/earn)
* Click the recurring payments card to load the splash page (/earn/payments/)
* Click the CTA button and confirm that the process to connect to Stripe is started.
